pages/configuring_webif.rst

```diff
@@ -308,106 +308,127 @@ If you want to run a load balancer/reverse proxy in front of Graylog, you need t
   - You are either using only HTTP or only HTTPS (no mixed content) for both the web interface endpoint and the REST API endpoint.
   - If you use SSL, your certificates must be valid and trusted by your clients.
 
-To help you with your specific environment, here are some example configurations for common scenarios:
+.. NOTE:: To help you with your specific environment, we have some example configurations. We take the following assumption in all examples. Your Graylog server.conf has the following settings set ``rest_listen_uri = http://127.0.0.1:12900/`` and ``web_listen_uri = http://127.0.0.1:9000/``. Your URL will be ``graylog.example.org`` with the IP ``192.168.0.10``.
 
 
-Using a Layer 3 load balancer (forwarding TCP Ports):
+Using a Layer 3 load balancer (forwarding TCP Ports)
 -----------------------------------------------------
```

Comment:
The length of the underline also has to be adapted. (Yes, Sphinx can be anal about such things.)